distribution/server/pom.xml

```diff
@@ -95,8 +95,8 @@
     </dependency>
 
     <dependency>
-      <artifactId>log4j</artifactId>
-      <groupId>log4j</groupId>
+      <groupId>org.apache.logging.log4j</groupId>
```

Comment:
For this we need the read depedency, since this determines what goes inside the distribution tar

Reply:
Thanks for the review.
I'm just trying to understand that how the legacy log4j dependency is used. The [log4j-1.2-api as part of Log4j2](https://logging.apache.org/log4j/2.x/log4j-1.2-api/index.html) provides the legacy log4j API but routes the log events to log4j2. I'd assume that this would be the correct approach if legacy log4j and log4j2 co-exist. 
However is legacy log4j even needed? I see that legacy log4j is excluded from many dependencies. `mvn dependency:tree` didn't give a direct answer, but I can try to investigate by temporarily removing the exclusions locally.

Reply:
Oh, I got confused with log4j2. I think here we're adding this dependency because of ZK. Your change is probably the correct one then.